src/common/libterminus/terminus.c

```diff
@@ -207,6 +212,11 @@ terminus_session_create (struct flux_terminus_server *ts,
     if (!(s->pty = flux_pty_open ()))
         goto error;
     flux_pty_set_flux (s->pty, ts->h);
+    flux_pty_aux_set (s->pty, "terminus_session", s, NULL);
```

Comment:
there's a malloc inside `flux_pty_aux_set()` so check for failure?

Reply:
Good catch, thanks!